core/src/main/java/nu/yona/server/subscriptions/entities/User.java

```diff
@@ -53,13 +53,13 @@ public static UserRepository getRepository()
 
 	private boolean isCreatedOnBuddyRequest;
 
-	@OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER, orphanRemoval = true)
+	@OneToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY, orphanRemoval = true)
```

Comment:
Despite the documentation, I can't remove all occurrences, as that causes the decryption check to fail. I've reverted this change, so all ``FetchType.LAZY`` instances are still in.